### PR TITLE
169036921 io utils integration

### PR DESCRIPTION
**Pull request by d-al-ibm**
added package and scripts for io-utils
added first integration with auto generated classes in registration step one
fixed problems with types in registration step one
updated README

### CODE REVIEW

File: src/components/Registration/RegistrationStepOne/SearchInstitutions.tsx [hunk 1]
```diff
@@ -4,32 +4,42 @@ import { AsyncTypeahead } from "react-bootstrap-typeahead";
...
-  pecs: ReadonlyArray<string>;
-  selectedPecIndex: number | null;
-  scope: string | null;
-}
+/*
+ * Defined a type for selected institution that will be filled with the form
+ * it holds every property of FoundNotRegisteredAdministration and all the properties of
+ * FoundRegisteredAdministration not shared with FoundNotRegisteredAdministration, as optionals
+ */
+type CustomInstitution = FoundNotRegisteredAdministration &
```
[the remark this answers is not in the document]
Comment (d-al-ibm) [reply src/components/Registration/RegistrationStepOne/SearchInstitutions.tsx]: yes, on the front end code I was using only institution but with io-utils/backend integration now there are classes named Administration
I will switch to use only administration

File: src/components/Registration/RegistrationContainer.tsx [hunk 2]
```diff
@@ -159,17 +162,17 @@ export const RegistrationContainer = withRouter(props => {
           <RegistrationStepOne
             onPecCheckboxChange={handlePecCheckboxChange}
             onScopeCheckboxChange={handleScopeCheckboxChange}
-            institutions={institutions}
-            onInstitutionSearch={handleIntitutionSearch}
-            onInstitutionSelected={handleInstitutionSelected}
-            selectedInstitution={selectedInstitution}
+            administrations={administrations}
+            onAdministrationSearch={handleIntitutionSearch}
```
Comment (gunzip) [inline src/components/Registration/RegistrationContainer.tsx]: handleAdminstrationSearch ? (there's a typo anyway)

Comment (d-al-ibm) [reply src/components/Registration/RegistrationContainer.tsx]: fixed

Comment (d-al-ibm) [reply src/components/Registration/RegistrationContainer.tsx]: the typo prevented me to find the string to substitute

**Pull request merged**